SwiftGit2Tests/RepositorySpec.swift

```diff
@@ -23,7 +24,7 @@ class RepositorySpec: QuickSpec {
 				let url = URL(fileURLWithPath: "blah")
 				let result = Repository.at(url)
 				expect(result.error?.domain) == libGit2ErrorDomain
-				expect(result.error?.localizedDescription).to(match("Failed to resolve path"))
+				expect(result.error?.localizedDescription.lowercased()).to(beginWith("failed to resolve path"))
```

Comment:
Perhaps something like

```swift
expect(result.error?.localizedDescription).to(match("(F|f)ailed to resolve path.*"))
```

would be a bit more tolerating to different libgit2 versions?

Reply:
Addressed in commit  https://github.com/SwiftGit2/SwiftGit2/pull/138/commits/43fa649e00ebe4b458b184103ad681af96b4a7dd